app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains
```

Comment:
dùng i18n có truyền biến và hiển thị số ít số nhiều trong i18n để khai báo 1 chuỗi hoàn chỉnh
https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations
https://guides.rubyonrails.org/i18n.html#pluralization